### PR TITLE
{network}Add tests for two network commands

### PR DESCRIPTION
**Pull request by YuanyuanNi**
**Description**

Add tests for two network commands, below tests will be affected:
test_network_ag_auth_cert
test_network_app_gateway_waf_policy_with_application_gateway

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Review by wangzelin007 — APPROVED** [review]

**Comment by yonzhan** [comment]
network

File: scripts/ci/credscan/CredScanSuppressions.json [hunk 1]
```diff
@@ -367,7 +368,8 @@
     {
       "file": [
         "src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_private_endpoint_connection_acfr.yaml",
-        "src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_private_link_resource_acfr.yaml"
+        "src/azure-cli/azure/cli/command_modules/network/tests/latest/recordings/test_private_link_resource_acfr.yaml",
+        "src\\azure-cli\\azure\\cli\\command_modules\\network\\tests\\latest\\recordings\\test_network_ag_auth_cert.yaml"
```
Comment (jsntcy) [inline scripts/ci/credscan/CredScanSuppressions.json:372]: Add to a separate "file" section as it has different justification.

**Review by jsntcy — APPROVED** [review]

**Pull request merged**